# Contains 1 discrete values
int64 id

# Contains 3 real values of robot expression
float64 p_happy 
float64 p_sad 
float64 p_neutral
